Infrastructure/gpg-ipdeny/nginx.conf

```diff
@@ -0,0 +1,107 @@
+worker_processes 4;
+daemon off;
+
+error_log /dev/stderr;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  charset utf-8;
+
+  log_format access_json '{"logType": "nginx-access", '
+                         ' "remoteHost": "$remote_addr", '
+                         ' "user": "$remote_user", '
+                         ' "time": "$time_local", '
+                         ' "request": "$request", '
+                         ' "status": $status, '
+                         ' "size": $body_bytes_sent, '
+                         ' "referer": "$http_referer", '
+                         ' "userAgent": "$http_user_agent", '
+                         ' "requestTime": $request_time, '
+                         ' "httpHost": "$http_host"}';
+
+  access_log /dev/stdout access_json;
```

Comment:
Looks like these will just sit around on the instance and we don't send them anywhere? While I don't think it adds a lot to send the whole thing to logit, I would be very interested in stats on how many users we are denying access to - is that something we can pipe to another log stream and consider uploading to the cloud along with all the app server logs?